cpu/cpu.go

```diff
@@ -157,57 +150,56 @@ func (p *Plugin) GetMetricTypes(cfg plugin.ConfigType) ([]plugin.MetricType, err
 	mList := make(map[string]bool)
 	for _, namespace := range namespaces {
 		namespace = strings.TrimRight(namespace, string(os.PathSeparator))
-		metricType := plugin.MetricType{
-			Namespace_: core.NewNamespace(strings.Split(namespace, string(os.PathSeparator))...)}
-		ns := metricType.Namespace()
+		mt := plugin.Metric{
+			Namespace: plugin.NewNamespace(strings.Split(namespace, string(os.PathSeparator))...)}
```

Comment:
@rashmigottipati, it seems that there is some overhead in creating a metric namespace. Probably it is historical changes which are no longer needed. What I mean is:
-  there is no benefit of using this `prefix` as joined vendor, fs, Name - probably slice would be better or use these contents straightforward 
- creating metric in [line 145](https://github.com/intelsdi-x/snap-plugin-collector-cpu/pull/55/files#diff-8e127506a19bdf5c074e74e13ab9d1bbR145) together with [line 161](https://github.com/intelsdi-x/snap-plugin-collector-cpu/pull/55/files#diff-8e127506a19bdf5c074e74e13ab9d1bbR161) - I think this should be simplified based on cpu plugin is supported plugin, so it should be easy to manage that in the future.